src/tools/brush/ScissorsTool.js

```diff
@@ -0,0 +1,329 @@
+      svgCursor: scissorsFillInsideCursor,
+    };
+
+    super(props, defaultProps);
+
+    this._resetHandles();
+
+    //
+    // Touch
+    //
+    /** @inheritdoc */
+    this.postTouchStartCallback = this._startOutliningRegion.bind(this);
+
+    /** @inheritdoc */
+    this.touchDragCallback = this._setHandlesAndUpdate.bind(this);
+
+    /** @inheritdoc */
+    this.touchEndCallback = this._applyStrategy.bind(this);
+
+    //
+    // MOUSE
+    //
+    /** @inheritdoc */
+    this.postMouseDownCallback = this._startOutliningRegion.bind(this);
+
+    /** @inheritdoc */
+    this.mouseClickCallback = this._startOutliningRegion.bind(this);
+
+    /** @inheritdoc */
+    this.mouseDragCallback = this._setHandlesAndUpdate.bind(this);
+
+    /** @inheritdoc */
+    this.mouseMoveCallback = this._setHandlesAndUpdate.bind(this);
+
+    /** @inheritdoc */
+    this.mouseUpCallback = this._applyStrategy.bind(this);
+
+    this._changeStrategy = this._changeStrategy.bind(this);
+    this._changeStrategy();
+  }
+
+  /**
+   * Render hook: draws the Scissors's outline, box, or circle
+   *
+   * @param {Object} evt Cornerstone.event#cornerstoneimagerendered > cornerstoneimagerendered event
+   * @memberof Tools.ScissorsTool
+   * @returns {void}
+   */
+  renderToolData(evt) {
+    const eventData = evt.detail;
+    const { element } = eventData;
+    const color = toolColors.getColorIfActive({ active: true });
+    const context = getNewContext(eventData.canvasContext.canvas);
+    const handles = this.handles;
+
+    draw(context, context => {
+      if (handles.points.length > 1) {
+        for (let j = 0; j < handles.points.length; j++) {
+          const lines = [...handles.points[j].lines];
+          const points = handles.points;
+
+          if (j === points.length - 1) {
+            // If it's still being actively drawn, keep the last line to
+            // The mouse location
+            lines.push(this.handles.points[0]);
+          }
+          drawJoinedLines(
+            context,
+            element,
+            this.handles.points[j],
+            lines,
+            color
+          );
+        }
+      }
+    });
+  }
+
+  /**
+   * Sets the start handle point and claims the eventDispatcher event
+   *
+   * @private
+   * @param {*} evt // mousedown, touchstart, click
+   * @returns {Boolean} True
+   */
+  _startOutliningRegion(evt) {
+    const consumeEvent = true;
+    const element = evt.detail.element;
+    const image = evt.detail.currentPoints.image;
+    const emptyPoints = !this.handles.points.length;
+
+    if (!emptyPoints) {
+      logger.warn('Something went wrong, empty handles detected.');
+
+      return null;
+    }
+
+    if (emptyPoints) {
+      this.handles.points.push({
+        x: image.x,
+        y: image.y,
+        lines: [],
+      });
+      this.currentHandle += 1;
+    }
+
+    external.cornerstone.updateImage(element);
+
+    return consumeEvent;
```

Comment:
You can just return true here, since the event consumption isn't variable in this function. The doc API describes what the returned variable means, so I think there is no need for a named variable.

Reply:
Fixed.